DEPENDENCIES.md

```diff
@@ -69,15 +69,15 @@ brew remove eosio
 #### Ubuntu 18.04 Package Install
 
 ```sh
-wget https://github.com/dfuse-io/eos/releases/download/v2.0.5-dm-12.0/eosio_2.0.5-dm.12.0-1-ubuntu-18.04_amd64.deb
-sudo apt install ./eosio_2.0.5-dm.12.0-1-ubuntu-18.04_amd64.deb
+curl -s https://api.github.com/repos/dfuse-io/eos/releases/latest|grep "browser_download_url.*18.04_amd64.deb"|cut -d : -f 2,3|tr -d \"|wget --show-progress -O ./eosio-dm-latest-18.04.deb -qi -
+sudo apt install ./eosio-dm-latest-18.04.deb
 ```
 
 #### Ubuntu 16.04 Package Install
 
 ```sh
-wget https://github.com/dfuse-io/eos/releases/download/v2.0.5-dm-12.0/eosio_2.0.5-dm.12.0-1-ubuntu-16.04_amd64.deb
-sudo apt install ./eosio_2.0.5-dm.12.0-1-ubuntu-16.04_amd64.deb
+curl -s https://api.github.com/repos/dfuse-io/eos/releases/latest|grep "browser_download_url.*16.04_amd64.deb"|cut -d : -f 2,3|tr -d \"|wget --show-progress -O ./eosio-dm-latest-16.04.deb -qi -
```

Comment:
When I ran this on my Mac, I did not see any progress and in fact, it didn't work, the downloaded file was an HTML file.

```
$ curl -s https://api.github.com/repos/dfuse-io/eos/releases/latest\|grep "browser_download_url.*16.04_amd64.deb"|cut -d : -f 2,3|tr -d \"|wget --show-progress -O ./eosio-dm-latest-16.04.deb -qi -
./eosio-dm-latest-16.04.deb                   100%[===============================================================================================>]   1.45M  2.49MB/s    in 0.6s

$ file ./eosio-dm-latest-16.04.deb
./eosio-dm-latest-16.04.deb: HTML document text, UTF-8 Unicode text, with very long lines

$ cat ./eosio-dm-latest-16.04.deb
<html lang="en">
  <head>
  <title>Repositories - GitHub Docs</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="google-site-verification" content="OgdQc0GZfjDI52wDv1bkMT-SLpBUo_h5nn9mI9L22xQ" />
  <meta name="google-site-verification" content="c1kuD-K2HIVF635lypcsWPoD4kilo5-jA_wBFyT4uMY" />
....
```

Reply:
Looks like you added a trailing `\` after latest that is causing this issue. It works fine on Mac OS

Reply:
Indeed, not sure what happened, copied that verbatim from the diff to test ... weird!

Reply:
Ok my terminal is adding it automatically, because there is no space after the `|` so the terminal thinks it's part of the url hence why it's adding it.

To ensure nobody else get this, can you add space around all pipes character please.

Reply:
In all three changes